lib/ex_doc/formatter/html/search_data.ex

```diff
@@ -59,6 +31,34 @@ defmodule ExDoc.Formatter.HTML.SearchData do
     end)
   end
 
+  defp extra(%{url: url} = map) do
+    [encode("#{map.id}", map.title, :extras, url)]
```

Comment:
Sorry, I should have caught it the first time around, but I am thinking this likely should not show up on search? I.e. we should skip URLs here too.

Reply:
And let's add a test for whatever decision we pick here. :)

Reply:
Done 👍